ONSdigital/dp-frontend-filter-flex-dataset
# Feature/5679 migrate GetAreasType endpoint to GetPopulationAreaTypes

### What

Migrated `GetAreasType` on the dimensions client endpoint to `GetPopulationAreaTypes` on the population client

Resolves trello ticket [Update dp-frontend-filter-flex to use updated function to retrieve area-types](https://trello.com/c/A70dGKH6/5679-update-dp-frontend-filter-flex-to-use-updated-function-to-retrieve-area-types)

### How to review

- Sense check
- Tests pass
- _Optionally_ run the stack locally; using the [cantabular-import](https://github.com/ONSdigital/dp-compose/tree/main/cantabular-import) docker container. 
  - Create a flexible dataset
  - Go to a [dataset landing page](http://localhost:8081/datasets/cantabular-flexible-example/editions/2021/versions/1)
  - Click 'change'
  - Verify that the area type is still listed and can be changed
  - **OR** call me 🤙 and I'll show you 

### Who can review

Frontend go dev
